src/Donations/index.tsx

```diff
@@ -254,15 +260,15 @@ function DonationInfo() {
               )}
               {(projectDetails.purpose === "trees" ||
                 projectDetails.purpose === "conservation") &&
-                projectDetails.tpo && (
+                projectDetails.ownerName && (
                   <a
                     rel="noreferrer"
                     target="_blank"
-                    href={`https://www.trilliontreecampaign.org/t/${projectDetails.tpo.slug}`}
+                    href={`https://www.trilliontreecampaign.org/t/${projectDetails.id}`}
```

Comment:
@tejassonar you can remove this hyperlink.

Reply:
Done